src/components/log/QueryLog.tsx

```diff
@@ -305,7 +305,7 @@ const dnssecColor = [
   "orange",
   "red",
   "red",
-  "red"
+  "orange"
```

Comment:
Just as a point of making things clearer... Do you think you could change each line of this to have a comment describing which status DNSSEC status this maps to?

```suggestion
  "orange" //unknown
```

It may not be obvious to everyone!